test/functional/apps/home/_navigation.ts

```diff
@@ -87,9 +87,7 @@ export default function({ getService, getPageObjects }: FtrProviderContext) {
 
       // Navigating back from time settings
       await browser.goBack(); // undo time settings
-      await browser.goBack(); // undo automatically set config, should it be in the history stack? (separate issue!)
```

Comment:
I've removed that, since the push of the default app state to domain in no longer a separate history item, there's no use for a user to use the back button and only the url changes.